src/main/scala/org/broadinstitute/dsde/firecloud/service/UserService.scala

```diff
@@ -117,35 +117,25 @@ trait UserService extends HttpService with PerRequestCreator with FireCloudReque
         }
       }
     } ~
-    requireUserInfo() { userInfo =>
     pathPrefix("api") {
       path("profile" / "billing") {
         passthrough(UserService.billingUrl, HttpMethods.GET)
       } ~
       path("profile" / "billingAccounts") {
-        oauthParams { (state, _) => requestContext =>
+        get { requestContext =>
           val pipeline = authHeaders(requestContext) ~> sendReceive
-          val extReq = Get(UserService.billingAccountsUrl)
-          pipeline(extReq) onComplete {
-            case Success(response) if response.status == Forbidden =>
-              val tryParseScopes = Try {
-                response.entity.asString.parseJson.convertTo[ErrorReport].message.parseJson.convertTo[BillingAccountScopes]
-              }
-              tryParseScopes match {
-                case Success(scopes) =>
-                  // user does not have appropriate scopes.  Ask user to enable them via OAuth
-                  val redirectURI = HttpGoogleServicesDAO.getGoogleRedirectURI(state, "auto", Option(scopes.requiredScopes ++ HttpGoogleServicesDAO.userLoginScopes))
-                  requestContext.complete(Forbidden, ErrorReport(Forbidden, BillingAccountRedirect(redirectURI).toJson.toString))
-                case _ =>
-                  requestContext.complete(Forbidden, response.entity)
-              }
-            case Success(response) => requestContext.complete(response.status, response.entity)
-            case Failure(regrets) => respondWithErrorReport(regrets, requestContext)
+          pipeline(Get(UserService.billingAccountsUrl)) map { response =>
+            response.status match {
+              // The proxy would prevent a user from getting here if their account wasn't enabled,
+              // so a 403 from Rawls is effectively guaranteed to mean that the user hasn't enabled
+              // the billing accounts scope.
+              case StatusCodes.Forbidden =>
+                requestContext.complete(
+                  StatusCodes.Forbidden, Map("billingPermissionsRequired" -> true))
```

Comment:
I agree with @davidangb, the design was that rawls passes back the required scopes and then whoever is responsible should ask for those scopes. I don't know why that was dropped. Looking to the comment above, "effectively guaranteed," I don't believe you.